e2e/package.json

```diff
@@ -2,13 +2,12 @@
   "name": "e2e",
   "version": "1.0.0",
   "scripts": {
-    "test": "bazel test //..."
+    "test": "bazel test //... && bazel build //..."
```

Comment:
did you actually need this? `test` should include building everything buildable (unless you use `--build_tests_only`)

Reply:
No, didn't end up needing it